reference_system/README.md

```diff
@@ -0,0 +1,80 @@
+# reference_system_autoware
```

Comment:
```suggestion
# Reference System
```

Reply:
@LanderU this file is specific for the autoware version of a reference system....i.e. it defines all the nodes in the autoware reference system.

rather than rename this to remove `autoware` it might be better to move this to a specific folder for `autoware`, or rename it to `autoware-requirements.md` and then link to it from a base `reference_system/README.md`